XamlStyler.Mac/Properties/Manifest.addin.xml

```diff
@@ -1,36 +1,44 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <ExtensionModel>
-	<Runtime>
-    	<Import assembly="Newtonsoft.Json.dll"/>
-    	<Import assembly="XamlStyler.Core.dll"/>
-    	<Import assembly="XamlStyler.Mac.dll"/>
-	</Runtime>
-	<Dependencies>
-		<Addin id="::MonoDevelop.Core" version="8.2.6" />
-		<Addin id="::MonoDevelop.Ide" version="8.2.6" />
-		<Addin id="::MonoDevelop.SourceEditor2" version="8.2.6" />
-	</Dependencies>
+    <Extension path="/MonoDevelop/Ide/StartupHandlers">
+        <Class class="Xavalon.XamlStyler.Mac.CommandHandlers.StartupCommandHandler" />
+    </Extension>
+    
     <Extension path="/MonoDevelop/Ide/MainMenu/Edit">
         <CommandItem id="Xavalon.XamlStyler.Mac.FormatXaml" />
     </Extension>
-    <Extension path="/MonoDevelop/SourceEditor2/ContextMenu/Editor">
+    <Extension path="/MonoDevelop/Ide/ContextMenu/DocumentTab">
         <CommandItem id="Xavalon.XamlStyler.Mac.FormatXaml" />
-	</Extension>
+    </Extension>
+    <Extension path="/MonoDevelop/TextEditor/ContextMenu/Editor">
+        <CommandItem id="Xavalon.XamlStyler.Mac.FormatXaml"/>
+    </Extension>
+    
     <Extension path="/MonoDevelop/Ide/ContextMenu/ProjectPad">
         <Condition id="ItemType" value="Project|Solution">
-            <CommandItem id="Xavalon.XamlStyler.Mac.FormatXamlBatch" />
+            <CommandItem id="Xavalon.XamlStyler.Mac.BatchFormatXaml" />
         </Condition>
     </Extension>
+    
     <Extension path="/MonoDevelop/Ide/Commands/Edit">
-        <Command id="Xavalon.XamlStyler.Mac.FormatXaml" _label="Format Xaml" _description="Reformats the XAML document to be much more readable" defaultHandler="Xavalon.XamlStyler.Mac.FormatXamlHandler" macShortcut="Meta|Shift|I" />
-        <Command id="Xavalon.XamlStyler.Mac.FormatXamlBatch" _label="Batch Format Xaml" _description="Reformats all XAML documents to be much more readable" defaultHandler="Xavalon.XamlStyler.Mac.FormatXamlBatchHandler" />
+        <Command 
```

Comment:
Does this expose two options for the user to select between?

Reply:
These commands are quite different in behavior.

FormatXaml is command for formatting single Xaml document. It appears in 3 places:
- Text Editor => Right Click => Context Menu => Format Xaml
- Text Editor => File Tab => Right Click => Context Menu => Format Xaml
- Toolbar => Edit Menu => Format Xaml

FormatXamlBatch is command for formatting all Xaml files inside specific project/solution. It appears in 2 places:
- Solution Pad => Project Item => Right Click => Batch Format Xaml
- Solution Pad => Solution Item => Right Click => Batch Format Xaml

Reply:
Added explanation with screenshots in message below